src/containers/underthehood/components/Web3ConnectionInfo.js

```diff
@@ -15,7 +17,7 @@ export default class Web3ConnectionInfo extends React.Component {
     }
 
     render() {
-        const { isLoading, isConnected, web3Instance, network, error, info } = this.props.web3Connect;
+        const { isLoading, isConnected, web3Instance, error, info } = this.props.web3Connect;
         const stringify = stringifier({ maxDepth: 3, indent: "   " });
```

Comment:
`stringify` and `stringifyInfo` are the same thing.

Reply:
done